pkg/karmadactl/join/join.go

```diff
@@ -98,9 +99,13 @@ func (j *CommandJoinOption) Complete(args []string) error {
 		j.ClusterName = args[0]
 	}
 
-	// If '--cluster-context' not specified, take the cluster name as the context.
+	// If '--cluster-context' not specified, use current context.
 	if len(j.ClusterContext) == 0 {
-		j.ClusterContext = j.ClusterName
+		apiconfig, err := clientcmd.LoadFromFile(j.ClusterKubeConfig)
```

Comment:
It seems that the default context is exactly current context when `j.ClusterContext = ""`.

Reply:
I just tested and looked at the following code. When the context is empty, use the current context.